src/test/resources/config/reportPortal.properties

```diff
@@ -0,0 +1,4 @@
+browserName=chrome
+login.url=http://localhost:8080/ui/#login
+username=illiakosholapenko
+password=123456789
```

Comment:
don't store sensitive data in committed code

Reply:
It is tempopary option before before integration with jenkins is set. Will be fixed lately in course.

Reply:
it is prohibited to store sensitive data like username, password, keys, etc in the repository due to security reasons. Please, remove at least password